src/App/Page.php

```diff
@@ -130,7 +130,7 @@ public function logRuntime(IManageConfigValues $config, string $origin = '')
 	 * The return value will be casted to boolean if non-boolean was returned.
 	 * @since 5.0.0
 	 */
-	public function offsetExists($offset): bool
+	public function offsetExists(mixed $offset): bool
```

Comment:
Thanks for the edit, unfortunately the minimum PHP version we support is still PHP 7.3 (per `composer.json`) where this type-hint doesn't exist yet. Please remove any other references to this type.